### DISCUSSION TITLE
Ornaments...

### DISCUSSION THREAD

**Discussion by afarlie** [psb1558/Junicode-font#131] [Ideas]
For references I compiled a list of the ornaments in Junicode Two Beta -

[User:ShakespeareFan00/Junicode Ornum](https://en.wikisource.org/wiki/User:ShakespeareFan00/Junicode_Ornum)

You may wish to include something like it in the font documentation at some point.

Were these derived from the ones in :  https://archive.org/details/specimenofprinti00caslrich?

**Comment by psb1558** [comment]
Thanks, @afarlie! That's very useful. The ornaments are from a Caslon specimen, but I no longer know if it was that very one.

**Comment by afarlie** [comment]
Wikimedia Commons also has some other ornaments and Fluerons... but most date from works in later periods.

I will also note in passing 76 images of Aldus leaf ornaments on Wikimedia Commons, ;) (Most seem to be color variants though as opposed to different glyphs.)

https://commons.wikimedia.org/wiki/Category:Aldus_leaf

**Reply by psb1558** [reply]
I added the Caslon fleurons in a fit of whimsy, but now, alas, I seem to be all whimsied out.
